_scss/components/_header.scss

```diff
@@ -90,6 +93,11 @@
   @include u-height(10);
 }
 
+.usa-navbar {
+  height: 5rem;
+}
```

Comment:
Is this still necessary?

Reply:
It is not, that is now set in `_height.scss`